app/services/twitter_service.rb

```diff
@@ -1,5 +1,37 @@
 module TwitterService
+
+  class YTTweeter
+    def initialize(service,hangout)
+      @service = service
+      @host = hangout.broadcaster ? hangout.broadcaster.split[0] : 'Host'
```

Comment:
great that we have a more robust hangout.broadcaster, but the new default of "Host just hosted an online #scrum ..." feels a little strange - maybe change to "AV" or "AgileVentures" ...